Tasks/code-push-vsts-task/task.json

```diff
@@ -0,0 +1,117 @@
+{
+    "id": "b69d19a0-7f6d-11e5-a86a-c199aa7a6cf0",
+    "name": "CodePush",
+    "friendlyName": "CodePush",
+    "description": "A VSTS build task for releasing app updates to the CodePush service",
+    "author": "Microsoft Corporation",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "demands": [],
+    "version": {
+        "Major": "1",
+        "Minor": "0",
+        "Patch": "1"
+    },
+    "minimumAgentVersion": "1.83.0",
+    "instanceNameFormat": "Release $(packagePath) to CodePush ($(appName) - $(deploymentName)) ",
+    "inputs": [
+        {
+            "name": "authType",
+            "type": "pickList",
+            "label": "Authentication Method",
+            "defaultValue": "ServiceEndpoint",
+            "helpMarkDown": "",
+            "options": {
+                "ServiceEndpoint": "Service Endpoint",
+                "AccessKey": "Access Key"
+            }
+        },
+        {
+            "name": "serviceEndpoint",
+            "type": "connectedService:codepush-auth-key",
+            "label": "Service Endpoint",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The VSTS Service Endpoint that specifies the identity for Code Push.",
+            "visibleRule": "authType = ServiceEndpoint"
+        },
+        {
+            "name": "accessKey",
+            "type": "string",
+            "label": "Access Key",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Access key used to authenticate with the CodePush service",
+            "visibleRule": "authType = AccessKey"
+        },
+        {
+            "name": "appName",
+            "type": "string",
+            "label": "App Name",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Name of the app you want to release an update for"
+        },
+        {
+            "name": "packagePath",
+            "type": "filePath",
+            "label": "Package Path",
+            "defaultValue": "",
```

Comment:
Can we rename this label to "Update Contents Path" since we renamed that parameter in the CLI?